app.ytmdesktop.ytmdesktop.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>app.ytmdesktop.ytmdesktop</id>
+  
+  <name>ytmdesktop</name>
+  <summary>Free cross platform Desktop Player for YouTube Music</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>CC-BY-NC-1.0</project_license>
```

Comment:
upstream say the project is CC0

https://github.com/ytmdesktop/ytmdesktop/blob/master/LICENSE.md